OracleDatabase/SingleInstance/dockerfiles/12.1.0.2/runOracle.sh

```diff
@@ -196,6 +198,13 @@ else
   echo "#####################################"
 fi;
 
+# Exiting the script without waiting on the tail logs
+if [ "$1" = "--nowait" ]; then
+   # Creating state-file for identifyig container of the prebuiltdb extended image
```

Comment:
Typo "identifying"

Reply:
Fixed this and other files from where it was copy-paste :)